src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.cs

```diff
@@ -65,4 +65,59 @@ public static void DefaultSettings_Tcp_Binary_Echo_RoundTrips_String()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+    
+
+    // Https: Client and Server bindings setup exactly the same using default settings.
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
+    [ConditionalFact(nameof(Root_Certificate_Installed),
+                     nameof(SSL_Available))]
+#endif
+    [OuterLoop]
+    public static void DefaultSettings_Https_Text_Echo_RoundTrips_String()
+    {
+#if FULLXUNIT_NOTSUPPORTED
+        bool root_Certificate_Installed = Root_Certificate_Installed();
+        bool ssl_Available = SSL_Available();
+
+        if (!root_Certificate_Installed ||
+            !ssl_Available)
```

Comment:
I'm trying to follow a consistent indentation style in both the ConditionalFact and the conditional 'if' code.  Speaking as one who has looked through this ConditionalFact code a lot lately, it gets confusing if they don't all share a consistent "look".